package.json

```diff
@@ -58,13 +58,15 @@
   ],
   "scripts": {
     "suffixup": "curl -o public_suffix_list.dat https://publicsuffix.org/list/public_suffix_list.dat && ./generate-pubsuffix.js",
-    "test": "vows test/*_test.js"
+    "test": "vows test/*_test.js",
+    "coverage": "node node_modules/istanbul/lib/cli.js cover vows test/*_test.js"
```

Comment:
I think that requires a global install of istanbul, which is what I was trying to avoid. Is there another way to get an alias in the local project just through NPM and Node? I'm open to your suggestion; I was just trying to avoid errors on initial run of "npm run coverage".